### PR TITLE
Don't log skin downloading progress

### PR DESCRIPTION
**Pull request by def-**
since it's not really interesting for most players, as long as things work fine

### CODE REVIEW

File: src/engine/client/http.h [hunk 1]
```diff
@@ -55,7 +56,7 @@ class CRequest : public IJob
 	int RunImpl(CURL *pHandle);
 
 public:
-	CRequest(const char *pUrl, CTimeout Timeout);
+	CRequest(const char *pUrl, CTimeout Timeout, bool LogProcess = true);
```
Comment (heinrich5991) [inline src/engine/client/http.h:59]:
```suggestion
	CRequest(const char *pUrl, CTimeout Timeout, bool LogProgress = true);
```

File: src/engine/client/http.h [hunk 2]
```diff
@@ -95,7 +96,7 @@ class CGetFile : public CRequest
 	IOHANDLE m_File;
 
 public:
-	CGetFile(IStorage *pStorage, const char *pUrl, const char *pDest, int StorageType = -2, CTimeout Timeout = CTimeout{4000, 500, 5});
+	CGetFile(IStorage *pStorage, const char *pUrl, const char *pDest, int StorageType = -2, CTimeout Timeout = CTimeout{4000, 500, 5}, bool LogProcess = true);
```
Comment (heinrich5991) [inline src/engine/client/http.h:99]:
```suggestion
	CGetFile(IStorage *pStorage, const char *pUrl, const char *pDest, int StorageType = -2, CTimeout Timeout = CTimeout{4000, 500, 5}, bool LogProgress = true);
```

**Review by heinrich5991 — APPROVED** [review]

File: src/engine/client/http.h (hunk 1 above)
Comment (def-) [reply src/engine/client/http.h:59]: Done.

File: src/engine/client/http.h (hunk 2 above)
Comment (def-) [reply src/engine/client/http.h]: Done.

**Comment by heinrich5991** [comment]
bors r+

Thanks.

**Pull request merged**